### ISSUE TITLE
Add support for "cycleway:both" tags

### ISSUE THREAD

**Issue by FFMbyBicycle** [abrensch/brouter#414]
Currently we can only check for "cycleway=both", but not the way more detailed "cycleway:both=*" tagging:

https://github.com/abrensch/brouter/blob/master/misc/profiles2/lookups.dat#L313-L333

There are more than half a million used tags and it would be nice if we could use them: https://taginfo.openstreetmap.org/keys/cycleway%253Aboth

If somebody tells me how to add new tags, I will push some PR for this and i.e. #398.

**Comment by EssBee59** [comment]
I was interested to estimate the impact of the "cycleway:both" tag, here the results for Hessen/Germany (got with overpass-turbo):

total occurences:  18.547

number of occurences by value:
=lane                    671
=no                      17673
=none                  5
=separate            41
=shared_lane      54
=track                 99
=yes                   4

Yes.it would be nice if we could use the tag in scripts (by nearly 900 highways the existing cycleway are not visible currently)

**Comment by afischerdev** [comment]
I agree, this would a nice feature. 
The bicycle streets exploding in my area as well.

Let us have a look if there are some other aspects for lookups.dat (like #398 or #402) and then start an update.

I could collect the lookups data and make a suggestion.
Is there someone who like to do the update on the standard profiles?

**Comment by polyscias** [comment]
I am not so sure I would like to have an update of the profiles that somehow prefer certain cycleway:both values. I have seen figures telling that roads with cycleways (apart from tracks) are no safer than the simular way without cycleway tracks. If anything is important it is the width.

> Let us have a look if there are some other aspects for lookups.dat (like #398 or #402) and then start an update.

Would be good to start a new issue for this and gather there all information I think.
I like to review the current lookup.dat values, I think some can be removed, but let me provide data for that.

**Comment by EssBee59** [comment]
Hello Polyscias,
The differences between "cycleway=" and "cycleway:booth=" are very very low...
Do you consider "cycleway=" currently in your biking scripts?
If roads with "cycleway" are not safer than similar way without cycleway, then I suspect a mapping error / issue.

Not every biker is using the cycleways (a.e.  bikers in group at week end) but others (bikers riding "alone" middle in the week on roads with trucks!) prefer safe lanes or tracks on the side of the road.. 
If the change is accepted, my intention is to consider "cyclewa:both" exactly as "cycleway"

**Comment by polyscias** [comment]
Yes, the differences between cycleway= and cycleway:both= are small, so agree, if cycleway= is taken into account, so should cycleway:both= be taken into account.

The profiles I use use only cycleway=opposite and it's variants (outdated) as alias for oneway:bicycle=no and I have some code that makes the profile handle cycleway=track similar as highway=cycleway.

> If roads with "cycleway" are not safer than similar way without cycleway, then I suspect a mapping error / issue.

Do you think a cycle lane of 50 cm with cars parked on the right is safer then no cycle lane? My point is that OSM is mapping all kind of attributes that are, at least for me, not important, what I think is important is width, if cars are parked parallel and the speed of car traffic.

I think a generic issue can be used to summarize things, for longer discussion it is likely better to use a separate topic.
